source/manual/govuk-app-domain-handling-during-migration.html.md

```diff
@@ -5,18 +5,26 @@ section: Infrastructure
 layout: manual_layout
 type: learn
 parent: "/manual.html"
-last_reviewed_on: 2019-10-21
+last_reviewed_on: 2020-01-23
 review_in: 3 months
 ---
 
-The app-by-app migration plan to move GOV.UK to AWS introduced the need to introduce an `$app_domain_internal` parameter in addition to the `$app_domain` used prior. This is due to the fact that in AWS, we use a `<environment>.govuk-internal.digital` domain in addition to the `<environment>.govuk.digital` domain.
+The app-by-app migration plan to move GOV.UK to AWS introduced an `$app_domain_internal` parameter in addition to the `$app_domain` used previously. 
 
-Furthermore, the `app_domain` parameter may be set to the `<environment>.publishing.service.gov.uk` for migrated apps as well. The exact configration depends on the current state of the migrated app as well as its dependencies. For example, migrated backend applications, such as Support, may be configured to use the `<environment>.publishing.service.gov.uk` `$app_domain` to facilitate access to Signon over the internet. Applications in Carrenza which talk to AWS over the VPN need to be able to resolve the `<environment>.govuk-internal.digital`. This internal domain is resolved to the private IP CIDR in AWS (at the moment this is only done for RabbitMQ exchange federation).
+This is necessary because in AWS we use a `<environment>.govuk-internal.digital` domain in addition to the `<environment>.govuk.digital` domain.
+
+Furthermore, the `app_domain` parameter may be set to the `<environment>.publishing.service.gov.uk` for migrated apps as well. The exact configuration depends on the current state of the migrated app and its dependencies. 
+
+For example, migrated backend applications, such as Support, may be configured to use the `<environment>.publishing.service.gov.uk` `$app_domain` to facilitate access to Signon over the internet. 
+
+Applications in Carrenza which talk to AWS over the VPN need to resolve `<environment>.govuk-internal.digital`. This internal domain is resolved to the private IP CIDR in AWS (at the moment this is only done for RabbitMQ exchange federation).
 
 As a rule of thumb:
 
-- Applications which have been moved to AWS and have all their dependencies in AWS will use `$app_domain=<environment>.govuk.digital` and `<$app_domain_internal=<environment>.govuk-internal.digital` consistently.
+- Applications which have been moved to AWS and have all their dependencies in AWS will use `$app_domain=<environment>.govuk.digital` and `$app_domain_internal=<environment>.govuk-internal.digital`.
 - Applications which remain in Carrenza, including all their dependencies, will only use `app_domain=<environment>.publishing.service.gov.uk`.
 - Applications having dependencies in both AWS and Carrenza will require some customisation of service resolution in form of a Plek URI override and may use either `$app_domain=<environment>.govuk.digital` or `$app_domain=<environment>.publishing.service.gov.uk`.
```

Comment:
What's a Plek URI override? Do we have existing documentation on that to link to? We mention it twice but never say what it actually means. (Most of us probably know, but being explicit is better.)

Reply:
We don't have docs on Plek override in `govuk-developer-docs` but it's referenced in code: https://github.com/alphagov/plek/blob/master/lib/plek.rb#L27